pkgs/servers/home-assistant/custom-components/daikin_onecta/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "jwillemsen";
+  domain = "daikin_onecta";
+  version = "4.2.2";
+
+  src = fetchFromGitHub {
+    owner = "jwillemsen";
+    repo = "daikin_onecta";
+    tag = "v${version}";
+    hash = "sha256-kjFBy+Nq9aQSsmvzWT2Wy/BwMpyd8GdgSXIrp44cMnA=";
+  };
+
+  dontBuild = true;
```

Comment:
The builder defaults to `true`.

```suggestion
```

Reply:
I only tested this against 24.11, where this was required, thanks for pointing it out!